doc/BoxPlatform.md

```diff
@@ -1,71 +1,133 @@
+[request performRequestWithCompletion:^(BOXUser *user, NSError *error) {
+    if (error) {
+        NSLog(@"Error getting user info");
+    } else {
+        NSLog(@"Retrieved user with login: %@", user.login);
+    }
+}];
+```
 
-    _boxClient = [BOXContentClient clientForServerAuthUniqueId:@"A_UNIQUE_ID"
-                                                  initialToken:nil
-                                           fetchTokenBlockInfo:nil
-                                               fetchTokenBlock:fetchTokenBlock];                                                          
+__Swift:__
+```swift
+let fetchToken: ServerAuthFetchTokenBlock = { (
+        uniqueID: String?,
+        fetchTokenBlockInfo: [AnyHashable: Any]?,
+        completion: ((String?, Date?, Error?) -> Void)!
+    ) -> Void in
     
-    //Example of making an API call
-    BOXUserRequest *request = [_boxClient currentUserRequest];
-    [request performRequestWithCompletion:^(BOXUser *user, NSError *error) {
-        if (error){
-            NSLog(@"Error getting user info");
+    // NOTE: This code is specific to your app; it is responsible for retrieving new tokens from a secure server
+    NSLog("Attempting to retrieve new access token from server for unique ID: \(uniqueID)");
+    
+    // Make a request to the secure sever for a token
+    let url = URL(string: "https://YOUR_SECURE_SERVER.example.com/token?id=\(uniqueID ?? "")")!
+    let task = URLSession.shared.dataTask(with: url) { data, response, error in
+        
+        if error != nil {
+            NSLog("Failed to retrieve new access token from server")
+            completion(nil, nil, error)
+            return
         }
-        else {
-            NSLog(@"Retrieved app user info. Login=%@", user.login);
+        
+        guard let httpResponse = response as? HTTPURLResponse, httpResponse.statusCode == 200 else {
+            NSLog("Server returned incorrect response")
+            completion(nil, nil, Error("Token sever returned incorrect response"))
+            return
         }
-    }];
-
-    return YES;
+        
+        let token = String(data: data!, encoding: .utf8)!
+        NSLog("Succeeded in retrieving new access token: \(token)")
+        
+        // Pass token and expiration time back to the client
+        completion(token, Date(timeIntervalSinceNow: 2700), nil)
+    }
+    task.resume()
 }
 
-ServerAuthFetchTokenBlock myFetchTokenBlock = ^(NSString *uniqueId, NSDictionary *fetchTokenBlockInfo, void (^completion)(NSString *, NSDate *, NSError *))
-{
-    //this code is specific to your app; it is responsible for retrieving new tokens from a secure server
-    NSLog(@"Attempting to retrieve new access token from server for uniqueId: %@", unique);
+// Create a client for the authenticated user; the client is keyed to that user's unique ID for caching purposes
+// The unique ID can be any identifier for the user, or a dummy value
+let user = ServerAuthUser(uniqueID: "UNIQUE_ID")!
```

Comment:
It's probably clear, but maybe call this `authUser` or something, as `user` gets shadowed in the block below.